cluster-autoscaler/config/migconfig.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+type MigConfig struct {
+	MinSize int
+	MaxSize int
+	Url     url.URL
+}
+
+type MigConfigFlag []MigConfig
+
+func (this *MigConfigFlag) String() string {
```

Comment:
Can we just use combination of `fmt.Sprintf` and `strings.Join`? It'll be much more readable.